lockr/lockr.d.ts

```diff
@@ -0,0 +1,113 @@
+// Type definitions for lockr 0.8.3
+// Project: https://github.com/tsironis/lockr
+// Definitions by: Dror Weiss <https://github.com/droritos>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+declare var Lockr: lockr.ILockrStatic;
+
+declare module lockr {
+    interface ILockrStatic {
```

Comment:
http://www.typescriptlang.org/Handbook#writing-dts-files

> Naming Conventions
In general, do not prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.